src/docs/releases/1.9.0.md

```diff
@@ -70,6 +70,10 @@ Additionally, if you need to enable indexing for text file with `.txt`, `.md` ex
 
 If you need to enable indexing for other extensions like (`.docx`, or `.pptx`), you will need the `OrchardCore.Media.Indexing.OpenXML` feature.
 
+### Email Module
+
+In the past, we use `Body` and `BodyText` to distinguish between the `MailMessage` body format, in this release `MailMessageBody` has been used instead.
```

Comment:
```suggestion
Previously, emails sent from Orchard Core could have either a plain text body, or an HTML body, but not both. Now, they can have both. This also brings some code-level API changes, see below.

When interacting with email-related services from code, `MailMessage`, the class representing an e-mail, exposed a `string` `Body` property. This could contain either plain text or HTML, which was indicated by `IsHtmlBody`.

While this API is still available, it's deprecated in favor of the new `Body` property that uses the new `MailMessageBody` type. This can contain a plain text and/or HTML body.
```